src/language/language_type.tera.rs

```diff
@@ -307,6 +307,7 @@ impl LanguageType {
     ///
     /// assert_eq!(rust, Some(LanguageType::Rust));
     /// ```
+    #[must_use]
     pub fn from_file_extension(extension: &str) -> Option<Self> {
```

Comment:
The `#[must_use]` doesn't do anything here since `Option` is already `#[must_use]`

Reply:
I was not aware of that. It could have been a false positive in clippy. :)